sickbeard/name_parser/parser.py

```diff
@@ -463,13 +463,13 @@ def parse(self, name, cache_result=True):
         final_result.quality = self._combine_results(file_name_result, dir_name_result, 'quality')
 
         if not final_result.show:
-            raise InvalidShowException("Unable to match {} to a show in your database".format
-                                       (name.encode(sickbeard.SYS_ENCODING, 'xmlcharrefreplace')))
+            raise InvalidShowException("Unable to match {0} to a show in your database. Parser result: {1}".format
+                                       (name.encode(sickbeard.SYS_ENCODING, 'xmlcharrefreplace'), file_name_result or dir_name_result))
```

Comment:
This encoding isn't related to the PP encoding issues.  The idea is it's safer to log unicode messages so making the string unicode and skipping the encode is less prone to errors because python itself does the unicode coercion.  e.g. 
```python
u"Unable to match {0} to a show in your database. Parser result: {1}".format(
    name, file_name_result or dir_name_result)
```